<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PotdPreview</class>
 <widget class="QDialog" name="PotdPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Picture Of The Day Desciption Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Getting Picture Of The Day Preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="potdLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>157</width>
       <height>101</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="font_label">
           <property name="text">
            <string>Font to use:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="potdFontComboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string notr="true">DejaVu Sans</string>
           </property>
           <property name="maxVisibleItems">
            <number>10</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="currentFont">
            <font>
             <family>DejaVu Sans</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="descrPos_label">
           <property name="text">
            <string>Description position:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="potd_description_bottom_radioButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Bottom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="potd_description_top_radioButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Top</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="textColorPotd">
           <property name="minimumSize">
            <size>
             <width>159</width>
             <height>29</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Text Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backgroundColorPotd">
           <property name="minimumSize">
            <size>
             <width>158</width>
             <height>29</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Background Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="margins_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Margins</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="left_margin_label">
           <property name="text">
            <string>Left:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="left_margin_spinbox">
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="right_margin_label">
           <property name="text">
            <string>Right:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="right_margin_spinbox">
           <property name="maximum">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="bottom_top_margin_label">
           <property name="text">
            <string>Bottom:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="bottom_top_margin_spinbox">
           <property name="maximum">
            <number>200</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
